### ISSUE TITLE
Write an easy to understand guide for the practical implementation of your concepts

### ISSUE THREAD

**Issue by jalilahmed** [rotot0/tab-ddpm#12]
I would suggest to include a bare bone basic pipeline of your code. The current layout of the code is every difficult to understand with a lot of boilerplate useless coding in-between (which makes sense in API's) but is not really required to understand the concept.

**Comment by LordofUniverse** [comment]
Yeah I would request for the same. It would be really helpful for us to have a guide. The currect readme is still hard for us to interpret

**Comment by Chandan-IITI** [comment]
Even, it is difficult to run the code. Getting lots of errors.

**Comment by Oefuli** [comment]
If someone has own data: Can you please explain step by step what someone needs to prepare so that your algorithm can process someone's data?

**Comment by celsofranssa** [comment]
The author could include this approach as a synthesizer in [SDV](https://github.com/sdv-dev/SDV/issues/2315).
SDV is very simple to install and use, has excellent [documentation](https://docs.sdv.dev/sdv), and has a large community.
